server/svix-server/src/v1/endpoints/attempt.rs

```diff
@@ -162,10 +160,10 @@ async fn list_attempted_messages(
     )))
 }
 
-/// Additional parameters (besides pagination) in the query string for the "List Attempts by
-/// Endpoint" enpoint.
+/// Additional parameters (besides pagination, and one parameter for by message) in the query string
+/// for the "List Attempts by Endpoint" enpoint and "List Attempts by Msg" endpoint.
 #[derive(Debug, Deserialize, Validate)]
-pub struct ListAttemptsByEndpointQueryParameters {
+pub struct ListAttemptsByEndpointOrMsgQueryParameters {
```

Comment:
I think the query params are different for both of them, you shouldn't share the structure. They happen to be *mostly* similar (but not identical) now, but this is not necessarily going to be the case. It adds more confusing than clarity, I'd say.